Sources/SwiftWarningControl/WarningControlRegions.swift

```diff
@@ -201,6 +210,15 @@ extension WarningControlRegionTree {
   }
 }
 
+extension WarningControlRegionTree {
+  /// Produce a set of diagnostic group identifiers contained in
+  /// this tree which have a behavior specifier other than `ignored`.
+  @_spi(ExperimentalLanguageFeatures)
+  public func getEnabledDiagnosticGroups() -> Set<DiagnosticGroupIdentifier> {
```

Comment:
Should this just be a computed property? Or rather, why not expose `enabledGroups` as `@_spi(…) public private(set)`?

Reply:
Yeah, that makes sense. Changed to be a `@_spi(…) public private(set)` property.